SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -119,7 +120,8 @@ Interest InterestFactory::CreateServerWorkerInterest(const UAbstractLBStrategy*
 		ServerQuery.FullSnapshotResult = true;
 	}
 
-	if (SpatialGDKSettings->bEnableOffloading)
+	// TODO(harkness): Rethink this.
```

Comment:
As discussed, should be good to move this conditional's logic to the load balancing strategy